src/comm/AcceptLimiter.cc

```diff
@@ -24,20 +24,17 @@ Comm::AcceptLimiter::Instance()
 void
 Comm::AcceptLimiter::defer(const Comm::TcpAcceptor::Pointer &afd)
 {
-    ++ (afd->isLimited);
-    debugs(5, 5, afd->conn << " x" << afd->isLimited);
+    debugs(5, 5, "deferring " << afd->conn);
```

Comment:
Instead of printing `deferring` (which is redundant because Squid will print `defer` (i.e. the method name) automatically), please report the current queue size. For example:

```
debugs(5, 5, afd->conn << "; already queued: " << deferred_.size());
```